libs/components/popovers/src/lib/modules/dropdown/dropdown.component.html

```diff
@@ -1,63 +1,61 @@
 <div class="sky-dropdown">
-  <button
-    #triggerButton
-    class="sky-btn sky-dropdown-button"
-    type="button"
-    [attr.aria-expanded]="isOpen"
-    [attr.aria-controls]="isOpen ? menuId : null"
-    [attr.aria-haspopup]="menuAriaRole"
-    [attr.aria-label]="
-      label ||
-      (buttonType === 'context-menu'
-        ? (contentInfoObs | async)?.descriptor
-          ? (contentInfoObs | async)?.descriptor?.type === 'text'
-            ? ('skyux_dropdown_context_menu_with_content_descriptor_default_label'
-              | skyLibResources: (contentInfoObs | async)?.descriptor?.value)
-            : undefined
-          : ('skyux_dropdown_context_menu_default_label' | skyLibResources)
-        : undefined)
-    "
-    [attr.aria-labelledby]="
-      !label &&
-      buttonType === 'context-menu' &&
-      (contentInfoObs | async)?.descriptor?.type === 'elementId'
-        ? screenReaderLabelContextMenu.id +
-          ' ' +
-          (contentInfoObs | async)?.descriptor?.value
-        : undefined
-    "
-    [attr.title]="title"
-    [disabled]="disabled"
-    [ngClass]="[
-      'sky-dropdown-button-type-' + buttonType,
-      'sky-btn-' + buttonStyle
-    ]"
-  >
-    @switch (buttonType) {
-      @case ('context-menu') {
-        <sky-icon icon="ellipsis" />
-      }
-      @default {
-        <div class="sky-dropdown-button-container">
-          <div class="sky-dropdown-button-content-container">
-            <ng-content select="sky-dropdown-button" />
-          </div>
-          <div class="sky-dropdown-button-icon-container">
-            <sky-icon
-              *skyThemeIf="'default'"
-              class="sky-dropdown-caret"
-              icon="caret-down"
-            />
-            <sky-icon
-              *skyThemeIf="'modern'"
-              class="sky-dropdown-caret"
-              icon="chevron-down"
-            />
+  <ng-content select="[skyDropdownTrigger]">
+    <button
+      #triggerButton
+      class="sky-btn sky-dropdown-button"
+      skyDropdownTriggerBase
```

Comment:
I had to split the trigger directive out into two implemations, one that does not set ARIA label and one that does. The one that does not is used here and exists because moving the logic into a computed signal caused existing unit tests in other libraries to fail, which means our consumers would also likely have failures. The one that does set ARIA label is the one to be used outside of the dropdown component.